minvws/nl-kat-coordination
# Fix table in DNS Report

### Changes
This PR fixes the "Other findings found" table in the DNS Report.

### Issue link
Closes #3635

### Demo
![afbeelding](https://github.com/user-attachments/assets/6d88d3f7-0c35-46f2-b21f-23ae7d8218f7)

### QA notes
- Create a DNS Report
- Check the chapter "Other findings found"
- Findings in table should be readable, all occurences should be visible with title and description

---

### Code Checklist







---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
